packages/initiatives/src/initiatives.ts

```diff
@@ -1,47 +1,53 @@
 /* Copyright (c) 2018 Environmental Systems Research Institute, Inc.
  * Apache-2.0 */
 
-import { request, IRequestOptions, IParams } from "@esri/arcgis-rest-request";
+import {
+  request,
+  IRequestOptions,
+  IParams,
+  getPortalUrl
+} from "@esri/arcgis-rest-request";
 import { IItem } from "@esri/arcgis-rest-common-types";
 import { getItem, getItemData } from "@esri/arcgis-rest-items";
-import { IInitiative, IInitiativeItem } from "@esri/hub-common";
+import { IInitiativeModel, IInitiativeItem } from "@esri/hub-common";
 import { fetchDomain } from "@esri/hub-sites";
-
-export interface IInitiativeRequestOptions extends IRequestOptions {
-  /**
-   * Set this value to false to avoid making a web request to fetch the item's data.
-   */
-  data: boolean;
-}
+import { getProp, cloneObject } from "@esri/hub-common";
+import { migrateSchema, CURRENT_SCHEMA_VERSION } from "./migrator";
 
 /**
- * Get the initiative item + data in one call
+ * ```js
+ * // fetch initiative by id, along with the data
+ * fetchInitiative('3ef...', {data:true})
```

Comment:
I don't think that `{data:true}` is a valid options hash, right?

Reply:
eagle-eyes++